sdk/src/main/java/io/matchmore/sdk/MatchMore.kt

```diff
@@ -36,48 +36,142 @@ object MatchMore {
 }
 
 interface MatchMoreSdk {
+    /**
+     * Main mobile device created by `startUsingMainDevice()`
```

Comment:
I would go for something more exhaustive like: 
Main device is the device on which the app will be running.
Mobile device object is used to represent the user's smartphone.

Reply:
Let's change in iOS SDK, and then we will sync